Documentation/DocuBlocks/Rest/Documents/post_create_document.md

```diff
@@ -239,5 +244,27 @@ Use of returnNew:
     logJsonResponse(response);
     db._drop(cn);
 @END_EXAMPLE_ARANGOSH_RUN
+
+@EXAMPLE_ARANGOSH_RUN{RestDocumentHandlerPostOverwrite}
+    var cn = "products";
+    db._drop(cn);
+    db._create(cn, { waitForSync: true });
+
+    var url = "/_api/document/" + cn;
+    var body = '{ "Hello": "World", "_key" : "lock" }';
+    var response = logCurlRequest('POST', url, body);
+    // insert
+    assert(response.code === 201);
+    logJsonResponse(response);
+
+    url = "/_api/document/" + cn + "?overwrite=true";
+    response = logCurlRequest('POST', url, body);
```

Comment:
shouldn't you rather write another document with the same id here then completely the same again?

Reply:
The document has the "_key" already defined. Therefor posting the same document twice triggers the behavior that is meant to be shown.